knowledge-base/catch-all-exceptions-globally.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Catch all exceptions globally using ErrorBoundary
+description: How to use ErrorBoundary to catch all exceptions globally?
+type: how-to
+page_title: Catch all exceptions globally using ErrorBoundary
+slug: common-kb-catch-all-exceptions-globally
+position: 
+tags: telerik,blazor,catch,all,exceptions,globally,error,boundary
+ticketid: 
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>UI for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+Is it possible to catch all errors in Blazor Server like in MVC 5 in global.asax on Application_Error?
+
+We are trying to integrate `ErrorBoundary` component but it seems it does not catch exceptions thrown from Window component. What is the correct way of using ErrorBoundary for the Telerik components?
+
+## Solution
+
+As of .NET 6, you can use [`ErrorBoundary` component](https://docs.microsoft.com/en-us/aspnet/core/blazor/fundamentals/handle-errors?view=aspnetcore-6.0#error-boundaries) to catch exceptions.
+
+To do so, wrap the existing content in an `ErrorBoundary` component. It will render error UI when an unhandled exception is thrown. The app continues to function normally, but the error boundary handles the exceptions in the affected component.
```

Comment:
```suggestion
To do so, wrap the existing content in an `ErrorBoundary` component. It will render error UI when unhandled exceptions are thrown. The application continues to function normally, but the error boundary handles the exception in the affected component.
```